app/javascript/packs/components/AddNewRestaurant.jsx

```diff
@@ -0,0 +1,176 @@
+import React, { Component } from 'react';
+import Dialog from 'material-ui/Dialog';
+import FlatButton from 'material-ui/FlatButton';
+import MenuItem from 'material-ui/MenuItem';
+import Slider from 'material-ui/Slider';
+import Checkbox from 'material-ui/Checkbox';
+import { TextValidator, ValidatorForm, SelectValidator } from 'react-material-ui-form-validator';
+import PropTypes from 'prop-types';
+import { addRestaurant } from '../utils/api';
+
+class AddNewRestaurant extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      openDialog: false,
+      formData: {
+        name: '',
+        address: '',
+        cuisine: '',
+        max_delivery_time: 120,
```

Comment:
this number is used multiple times, extract to constant

Reply:
👍